ci/docker/ml.Dockerfile

```
@@ -13,6 +13,6 @@ WORKDIR /ray
 COPY . .
 
 # Install Ray
-RUN SKIP_BAZEL_BUILD=1 RAY_INSTALL_JAVA=0 bash --login -i -c -- "python3 -m pip install -e /ray/python/"
+RUN SKIP_BAZEL_BUILD=1 RAY_INSTALL_JAVA=1 bash --login -i -c -- "python3 -m pip install -e /ray/python/"
```

```suggestion
RUN SKIP_BAZEL_BUILD=1 RAY_INSTALL_JAVA=0 bash --login -i -c -- "python3 -m pip install -e /ray/python/"
```

Let's revert this, as explained above we don't want to build the Ray java jars. This will drastically increase build time.

---

RayDP needs java binding to work though. It creates java actors. Where should we specify this option then?